### PR TITLE
Updated the shortcode and markdown documentation

### PR DESCRIPTION
**Pull request by n-boshnakov**
**What this PR does / why we need it**:
This PR adds a warning against mixing HTML and markdown formatting and updates the shortcodes documentation. 

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement developer
NONE
```

### CODE REVIEW

File: website/documentation/contribute/20_documentation/25_markup/_index.md [hunk 1]
```diff
@@ -12,4 +12,7 @@ a publishing platform mechanisms, such as Hugo's layouts. Considering that the s
 might be published by multiple platforms you should be considerate in using markup that may bind it 
 to a particular one.
 
+You should also avoid mixing HTML and markdown formatting, since it won't render correctly when the
+site is built.
```
Comment (JordanJordanov) [inline website/documentation/contribute/20_documentation/25_markup/_index.md:16]: This is valid only within the shortcode

**Review by JordanJordanov — CHANGES_REQUESTED** [review]

File: website/documentation/contribute/20_documentation/25_markup/_index.md (hunk 1 above)
Comment (n-boshnakov) [reply website/documentation/contribute/20_documentation/25_markup/_index.md]: Fixed in commit

**Review by JordanJordanov — APPROVED** [review]
/lgtm

**Pull request merged**